transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystem15Parser.java

```diff
@@ -181,7 +168,7 @@ private void parseCM(XMLExtendedStreamReader reader, List<ModelNode> operations)
     }
 
     private void addAttribute(XMLExtendedStreamReader reader, ModelNode operation, SimpleAttributeDefinition attributeDefinition) throws XMLStreamException {
-        String value = rawAttributeText(reader, attributeDefinition.getXmlName(), null);
+        String value = rawAttributeText(reader, attributeDefinition.getXmlName(), attributeDefinition.getDefaultValue().asString());
```

Comment:
@ochaloup The parser can't do this. It needs to preserve what it read from the xml, not introduce default values into the management model that were not what the user put in the xml. The subsystem can use a default value from the AttributeDefinition to configure the runtime services, but what's stored in the management model should reflect the user provided configuration.

It appears this method is only called by the parseCM method, so I think just eliminating the 'else' block that throws missingRequired would be fine.